lib/src/widgets/edit_table_drop_down.dart

```diff
@@ -81,8 +77,7 @@ class _EditTableDropDownState extends State<EditTableDropDown> {
 
   Widget _getEditTableItem(
       EditTableEnum type, BuildContext context, bool isMobile) {
-    String value = "";
-    String imagePath = ImageConstant.kiInsertRowBelowPng;
+    String value = ""
 
     switch (type) {
       case EditTableEnum.insertRowAbove:
```

Comment:
**Suggestion:** Wrap the logic inside a function to improve readability and reusability.
```suggestion
    String getValueFromType(EditTableEnum type) {
      String value = "";
      switch (type) {
        case EditTableEnum.insertRowAbove:
          // logic here
          break;
        // other cases
      }
      return value;
    }

    String value = getValueFromType(type);
```